backend/penndata/management/commands/get_rodin_events.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+
+import requests
+from bs4 import BeautifulSoup
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from penndata.models import Event
+
+
+RODIN_EVENTS_WEBSITE = "https://rodin.house.upenn.edu"
+HEADERS = {
+    "User-Agent": "Mozilla/5.0 AppleWebKit/537.36 Chrome/91.0.4472.124 Safari/537.36"
+}
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.scrape_calendar_page(RODIN_EVENTS_WEBSITE + "/calendar")
+        now = timezone.localtime()
+        current_day, current_month, current_year = now.day, now.month, now.year
+        if current_day > 25:
```

Comment:
nit: I would suggest utilizing the `datetime` library to calculate next month's year and stuff for you. I don't know the precise syntax, but I'd imagine you can do something like:
```python3
from datetime import timedelta
curr = timezone.localtime()
next = timezone.localtime() + timedelta(months=1)
next_day, next_month, next_year = next.day, next.month, next.year
```
I guess don't quote me on this, I would suggest checking what this kind of code does for like months that don't have a 31'st, but I feel like this might be better in terms of readability / scalability